src/components/KeyDataAssets/KeyDataAssets.tsx

```diff
@@ -58,7 +58,7 @@ const KeyDataAssets = () => {
             alt="Key Data Assets"
             width={1000}
             height={600}
-            loading="lazy"
+            priority={openTab === currentUsecase.imgSrc}
```

Comment:
This is because of this feedback. 
<img width="842" alt="image" src="https://github.com/open-metadata/openmetadata-site/assets/105535990/fa29caec-cc69-46c7-b038-c0292881a26c">
I saw that with this priority, maybe this issue can be handle.
And `currentUsecase.imgSrc` is the image Src. Sorry, this is my mistake, we have to add the name here, instead of imgSrc. 
As in local, I am not able to see much difference.